src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -4,11 +4,15 @@ import MessagePage from '../../components/MessagePage/MessagePage';
 import JournalEntryCard from '../../components/JournalEntryCard/JournalEntryCard';
 import Box from '@mui/material/Box';
 import Typography from '@mui/material/Typography';
+import { useNavigate } from 'react-router-dom';
+import Button from '@mui/material/Button';
+import { PossibleRoutes } from '../../utils/constants';
 
 import './DashboardPage.css';
 
 export interface mockData {
     id: number;
+    userId: string;
```

Comment:
Definitely necessary, will be used to link the user and their entries on the back end